src/view/uniprotkb/components/ProteinOverviewView.tsx

```diff
@@ -42,7 +44,11 @@ export const ProteinOverview: FC<{
     },
     {
       title: 'Annotation score',
-      content: <Bubble value={annotationScore} />,
+      content: (
+        <AnnotationScoreDoughnutChart>
+          {annotationScore}
```

Comment:
Would it be cleaner and more succinct if `{annotationScore}` was passed as `prop` to `AnnotationScoreDoughnutChart` e.g.

    <AnnotationScoreDoughnutChart score={annotationScore} />

Reply:
Good point. Updated.